src/ActionMonitor/Monitors/Monitor.php

```diff
@@ -291,8 +292,12 @@ public function log_action( array $args ) {
 
 		}
 
-		// we've saved at least 1 action, so we should update
-		$this->action_monitor->schedule_dispatch();
+		// If $should_dispatch is not set to false, schedule a dispatch. Actions being logged that
+		// set $should_dispatch to false will be logged, but not trigger a webhook immediately.
+		if ( false !== $should_dispatch ) {
+			// we've saved at least 1 action, so we should update
+			$this->action_monitor->schedule_dispatch();
```

Comment:
I did a similar thing in my multi-user preview PR 😆 but I just checked if it's a preview instead of doing this. Probably I should use the API you're adding here. But we need to first check what the implications of your PR are for the new multi-user preview.. 😆 chicken and egg?